### PR TITLE
Bump ra-data-postgrest to 2.3.0

### PR DESCRIPTION
**Pull request by slax57**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -4776,6 +4787,15 @@ define-data-property@^1.0.1, define-data-property@^1.1.1:
     gopd "^1.0.1"
     has-property-descriptors "^1.0.0"
 
+define-data-property@^1.1.4:
```
Comment (djhi) [inline yarn.lock:4790]: Duplicated dep. See above

File: yarn.lock [hunk 2]
```diff
@@ -3974,6 +3974,17 @@ call-bind@^1.0.4, call-bind@^1.0.5:
     get-intrinsic "^1.2.1"
     set-function-length "^1.1.1"
 
+call-bind@^1.0.7:
```
Comment (djhi) [inline yarn.lock:3977]: Duplicated deps. See above

File: yarn.lock [hunk 3]
```diff
@@ -6116,6 +6148,17 @@ get-intrinsic@^1.2.0, get-intrinsic@^1.2.1, get-intrinsic@^1.2.2:
     has-symbols "^1.0.3"
     hasown "^2.0.0"
 
+get-intrinsic@^1.2.4:
```
Comment (djhi) [inline yarn.lock:6151]: Duplicated dep. See above

File: yarn.lock [hunk 4]
```diff
@@ -6417,6 +6460,13 @@ has-property-descriptors@^1.0.0:
   dependencies:
     get-intrinsic "^1.1.1"
 
+has-property-descriptors@^1.0.2:
```
Comment (djhi) [inline yarn.lock:6463]: Duplicated dep. See above

File: yarn.lock [hunk 5]
```diff
@@ -9551,12 +9601,12 @@ q@^1.5.1:
   resolved "https://registry.yarnpkg.com/q/-/q-1.5.1.tgz#7e32f75b41381291d04611f1bf14109ac00651d7"
   integrity sha512-kV/CThkXo6xyFEZUugw/+pIOywXcDbFYgSct5cT3gqlbkBE1SJdwy6UQoZvodiWF/ckQLZyDE/Bu1M6gVu5lVw==
 
-qs@^6.11.1:
-  version "6.11.2"
-  resolved "https://registry.yarnpkg.com/qs/-/qs-6.11.2.tgz#64bea51f12c1f5da1bc01496f48ffcff7c69d7d9"
-  integrity sha512-tDNIz22aBzCDxLtVH++VnTfzxlfeK5CbqohpSqpJgj1Wg/cQbStNAz3NuqCs5vV+pjBsK4x4pN9HlVh7rcYRiA==
+qs@^6.12.1:
```
Comment (djhi) [inline yarn.lock:9604]: Duplicated dep. See below

File: yarn.lock [hunk 6]
```diff
@@ -10275,6 +10325,18 @@ set-function-length@^1.1.1:
     gopd "^1.0.1"
     has-property-descriptors "^1.0.0"
 
+set-function-length@^1.2.1:
```
Comment (djhi) [inline yarn.lock:10328]: Duplicated dep. See above

File: yarn.lock [hunk 7]
```diff
@@ -10312,6 +10374,16 @@ side-channel@^1.0.4:
     get-intrinsic "^1.0.2"
     object-inspect "^1.9.0"
 
+side-channel@^1.0.6:
```
Comment (djhi) [inline yarn.lock:10377]: Duplicated dep. See above

**Review by djhi — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 5 above)
Comment (slax57) [reply yarn.lock:9604]: Can't dedupe this one because cypress asks for a `~6.10.3` version :/

**Review by djhi — APPROVED** [review]

**Pull request merged**